examples/openvino/image-classification/README.md

```diff
@@ -39,4 +39,46 @@ python run_image_classification.py \
     --save_total_limit 3 \
     --seed 1337 \
     --output_dir /tmp/beans_outputs/
-```
\ No newline at end of file
+```
+
+### Joint Pruning, Quantization and Distillation on Swin for food101
```

Comment:
Please apply the same documentation comments as for README for the audio case.

Reply:
We have added the time & accuracy results for image-classification QAT.

https://github.com/huggingface/optimum-intel/blob/e23af292bf1b3643489ef4f6d7ad9c93ec7e370b/examples/openvino/image-classification/README.md?plain=1#L44